src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -55,7 +56,9 @@ public static CommitContributionSummary aggregateCommitResults(
     }
 
     /**
-     * Calculates the contribution variance of all authors.
+     * Calculates the contribution variance of all authors across contributions made between {@code startDate}
```

Comment:
I think the first sentence can retain the original `Calculates the contribution variance of all authors within a date range.` All the remaining details can follow in additional sentences.